tests/basic/storage_pool_test.go

```diff
@@ -2053,4 +2055,471 @@ var _ = Describe("{VolUpdateAddDrive}", func() {
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			opts := make(map[string]bool)
+			opts[scheduler.OptionsWaitForResourceLeakCleanup] = true
+			for _, ctx := range contexts {
+				TearDownContext(ctx, opts)
+			}
+		})
+	})
+	JustAfterEach(func() {
+		defer EndTorpedoTest()
+		AfterEachTest(contexts, testrailID, runID)
+	})
+})
+
+var _ = Describe("{CreateSnapshotsPoolResize}", func() {
+	/*
+		Steps : Try pool resize when ot of snapshots are created on the volume
+
+	*/
+	var testrailID = 50652
+	// Testrail Description : Try pool resize when lot of snapshots are created on the volume
+
+	var runID int
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("CreateSnapshotsPoolResize", "Validate storage pool expansion when lots of snapshots present on the system", nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	var contexts []*scheduler.Context
+	var totalSnapshotsPerVol int = 60
+
+	snapshotList := make(map[string][]string)
+	var selectedNode node.Node
+	var pickNode string
+
+	// Try pool resize when ot of snapshots are created on the volume
+	stepLog := "should get the existing storage node and expand the pool by resize-disk"
+
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+
+		contexts = make([]*scheduler.Context, 0)
+
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("snapcreateresizepool-%d", i))...)
+		}
+		ValidateApplications(contexts)
+
+		var stNode node.Node
+
+		// Get List of Volumes presnet in the Node
+
+		stNodes := node.GetStorageNodes()
+		if len(stNodes) == 0 {
+			dash.VerifyFatal(len(stNodes) > 0, true, "Storage nodes found?")
+		}
+		log.InfoD("List of Nodes present %s", stNodes)
+
+		for _, each := range contexts {
+			Volumes, err := Inst().S.GetVolumes(each)
+			log.FailOnError(err, "Listing Volumes Failed ")
+
+			log.InfoD("Get all the details of Volumes Present")
+			for _, vols := range Volumes {
+				log.InfoD("List of Volumes to inspect %T , %s", vols, vols.ID)
+				volInspect, err := Inst().V.InspectVolume(vols.ID)
+				selectedNode := volInspect.ReplicaSets[0].Nodes
+				randomIndex := rand.Intn(len(selectedNode))
+				pickNode = selectedNode[randomIndex]
+
+				for _, n := range stNodes {
+					if n.Id == pickNode {
+						stNode = n
+					}
+				}
+				fmt.Printf("Volume Inspect Details %s", volInspect)
+				log.FailOnError(err, "Listing Volumes Failed ")
+				for snap := 0; snap < totalSnapshotsPerVol; snap++ {
+					uuidCreated := uuid.New()
+					snapshotName := fmt.Sprintf("snapshot_%s_%s", vols.ID, uuidCreated.String())
+					snapshotResponse, err := Inst().V.CreateSnapshot(vols.ID, snapshotName)
+					log.FailOnError(err, "error identifying volume")
+					snapshotList[vols.ID] = append(snapshotList[vols.ID], snapshotName)
+					sPrint := fmt.Sprintf("Snapshot %s created with ID %s", snapshotName, snapshotResponse.GetSnapshotId())
+					log.InfoD(sPrint)
```

Comment:
Can we just do this, no need to define it as a var before logging it, you already doing it so in many places in this PR
```
log.InfoD("Snapshot [%s] created with ID [%s]", snapshotName, snapshotResponse.GetSnapshotId())
```